### ISSUE TITLE
Problems creating & evolving low-mass, low-temperature models since r12115

### ISSUE THREAD

**Issue by esandford** [MESAHub/mesa#705]
Extremely brief summary: Low-mass (<~ 1.5 M_Jupiter) that successfully run in MESA r12115 do not successfully run in r24.03.01. For masses ~0.25-1.5 M_Jupiter, the create_initial_model step and early evolution are fine, but evolution grinds to a halt once the temperature of the outer layers drops below ~150K. For masses <~0.25, create_initial_model fails.

**Overview**
I have been trying to use MESA to model a suite of homogeneously mixed objects between 0.2 and 20 M_Jupiter to an age of 20 Gyr. I am sticking as close as possible to the settings enumerated in Paxton et al. 2013, in order to roughly reproduce their figure 1. In this paper, they initialize all of these models at a radius of 5 R_Jupiter, with Y=0.27, Z=0.02, and mixing_length_alpha=2. The models are not irradiated.

I am attempting to model the following grid of initial masses: 20.0, 14.0, 9.8, 6.9, 4.8, 3.4, 2.3, 1.6, 1.1, 0.82, 0.57, 0.4, 0.28, 0.2 M_Jupiter.

**[Mostly] successful runs in r12115**
In MESA r12115, I adapted the make_brown_dwarf test suite inlist to create this inlist: 
[r12115_inlist_create+evolve.txt](https://github.com/user-attachments/files/16549221/r12115_inlist_create%2Bevolve.txt)

This runs successfully for initial masses between 20 and 0.28 M_Jupiter. At 0.2 M_Jupiter, in order to evolve the model past O(1.e-3) years, I have to decrease the initial radius from 5.0 to 1.5 R_Jupiter. The model can then evolve to ~19.6 Gyr, at which point there is a 
```
stop because hit EOS limits -- max_logQ > logQ_limit
```
error.

**Problems in r24.03.01**
In r24.03.01, I also start from the make_brown_dwarf test suite example to create the following inlists (divided into creation and evolution steps):

[r24.03.01_inlist_create.txt](https://github.com/user-attachments/files/16549419/r24.03.01_inlist_create.txt)
[r24.03.01_inlist_evolve.txt](https://github.com/user-attachments/files/16549420/r24.03.01_inlist_evolve.txt)

These run successfully for initial masses between 20 and 1.6 M_Jupiter. For initial masses from 1.1 to 0.28 M_Jupiter, the r24.03.01_inlist_create step runs successfully, but then during r24.03.01_inlist_evolve, the timestep shrinks to tiny values before 20 Gyr of evolution can elapse. 

If I irradiate the models by adding a Jupiter-like irradiation term to r24.03.01_inlist_evolve:

```
column_depth_for_irradiation = 300d0
irradiation_flux = 5.0d4 ! erg/cm^2/s
```

then the models do successfully evolve to 20 Gyr. 

For initial mass 0.2 M_Jupiter, r24.03.01_inlist_create fails, throwing the following error:
```
failed in kap_get get_kap_from_rhoT
 temp too low in integration
```

On the MESA-users list, Frank Timmes suggested that this behavior may be related to known issues in the low-temperature EOS and suggested I raise it here.

**Comment by Debraheem** [comment]
>  evolution grinds to a halt once the temperature of the outer layers drops below ~150K.

I don't believe there are any opacity tables in MESA which operate this low. I believe the lowest is the Freedman tables which operate down to logT = 2.7, so 500 K is the lower limit for all models, below which there are no opacities available in MESA. You would need to extrapolate or do something interesting to get opacities for these low temperatures. 

I'm curious why this wasn't an issue in previous MESA versions. I don't believe MESA extrapolates the opacity tables beyond their lower boundaries by default?

**Comment by Debraheem** [comment]
This might explain why irradiation is solving your issue.

**Comment by fxt44** [comment]
i'll see if i can get 12115 rolling. my sense is the eos (a hacked version of helm that transitioned to an unionized gas) in these earlier versions kept the temperature (pressure) higher, thus not rolling off the opacity tables.  the net effect is mimicked by adding irradiation.
